app/views/owner/planets/index.html.erb

```diff
@@ -10,3 +10,11 @@
   <% end %>
 </ul>
 <%= link_to "Add a planet", new_planet_path %>
+<% if policy(@planet).update? %>
```

Comment:
Hey! Thanks for the pull request! This might break, since there is no @planet for this controller, so I think it's okay to delete lines 13-20 here

Reply:
Oh yeah, I was almost forgetting to delete this, thanks!!!

Reply:
Great!! Other than that, everything looks good!